server/routers/crud_router.js

```diff
@@ -23,7 +23,13 @@ exports.getRouter = function (options) {
   router.get('/all', (req, res) => {
     db.connection().then(connection => {
       connection.db().collection(options.collectionName).find().toArray().then(data => {
-        res.send(data)
+        if (options.mapping) {
```

Comment:
This should occur on all read requests, and therefore should not be called `mapping`. It's more of a preprocessing